index.html

```diff
@@ -6,6 +6,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>La grève, c’est cool</title>
+
+    <meta property="og:title" content="la grève, c’est cool" />
```

Comment:
```suggestion
    <meta property="og:title" content="La grève, c’est cool" />
```

(C'est plus joli avec une majuscule, je trouve.)